app/src/test/java/protect/card_locker/DatabaseTest.java

```diff
@@ -434,7 +435,8 @@ public void cardAddAndRemoveGroups() {
 
         assertEquals(2, DBHelper.getGroupCount(mDatabase));
 
-        Group group1 = DBHelper.getGroup(mDatabase, "one");
+        Group group1 = DBHelper.getGroupByName(mDatabase, "one");
+        Group group2 = DBHelper.getGroupByName(mDatabase, "two");
```

Comment:
We should also test if we can get a group by ID, as that's the actual canonical thing.